Makefile

```diff
@@ -50,6 +50,12 @@ build.public: deps
 server: deps
 	$(HUGO) server -s $(DOC_ROOT) --environment $(ENVIRONMENT)
 
+.PHONY: new
+new:
+	$(HUGO) new --kind section-bundle -s $(DOC_ROOT) $(filter-out $@,$(MAKECMDGOALS))
+%:
+	@:
```

Comment:
What's this for?

Reply:
https://stackoverflow.com/a/45003119/7400521